### DISCUSSION TITLE
Issues vs Discussions

### DISCUSSION THREAD

**Discussion by Quin452** [uxbox/uxbox#599] [Q&A] [answered]
To the PenPot team, what are the rules for posting where?

I've been looking over both, see if there's anything to learn/help with, and a lot of the stuff is duplicated.
I for one have asked in Discussions about Fonts, and there is also an Issue about it (form another member).

Running my own repo, I generally keep the Issues section for actual issues, but I don't know what is preferred here.

**Comment by niwinz** [comment, marked as answer]
We have not defined the main rules, but it is high priority for us. 

For now and from my perspective, if you have an issue or concrete feature, go to Issues. If you want to ask something, discuss something before creating enhacement or issue -> discussions; then if the result of discussion is an enhacement, just create enhacement linking to the appropriate discussion.
